presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -414,8 +310,196 @@ as a part of SQL query by appending name with the table.
     ---------+------------------------------------------------------------------------------+-------------+--------------+--------------------+-----------------------------+--------------------------+----------------------+------------------+-------------------------------------------+--------------------------------------------+--------------+---------------+-------------
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
+SQL Support
+-----------
+
+The Iceberg connector supports querying and manipulating Iceberg tables and schemas
+(databases). Here are the examples of SQL Operations supported from Presto :
+
+CREATE SCHEMA
+^^^^^^^^^^^^^^
+
+Create a new Iceberg schema named ``web`` that will store tables in an
+S3 bucket named ``my-bucket``::
+
+    CREATE SCHEMA iceberg.web
+    WITH (location = 's3://my-bucket/')
+
+CREATE TABLE
+^^^^^^^^^^^^^
+
+Create a new Iceberg table named ``page_views`` in the ``web`` schema
+that is stored using the ORC file format, partitioned by date and
+country::
+
+    CREATE TABLE iceberg.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country']
+    )
+
+Create a Iceberg table with Iceberg format version 2::
+
+    CREATE TABLE iceberg.web.page_views_v2 (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioning = ARRAY['ds', 'country'],
+      format_version = '2'
+    )
+
+INSERT INTO
+^^^^^^^^^^^^
+
+Insert data into ``page_views`` table::
+
+    INSERT INTO iceberg.web.page_views VALUES(TIMESTAMP '2023-08-12 03:04:05.321', 1, 'https://example.com', current_date, 'country');
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``::
+
+    CREATE TABLE iceberg.web.page_views_new AS SELECT * FROM iceberg.web.page_views
+
+SELECT
+^^^^^^
+
+SELECT table operations are supported for Iceberg format version 1 & version 2 in the connector::
+
+    SELECT * FROM iceberg.web.page_views;
+
+    SELECT * FROM iceberg.web.page_views_v2;
+
+.. note::
+
+    Iceberg Tables with format version 2 where delete files are present are not filtered in ``SELECT``
```

Comment:
Does it mean it gives wrong results? "are not filtered in" is kind of vague here. Can you use more precise words?

Reply:
Iceberg Tables with format version 2 where delete files are present are not filtered in SELECT operations in Presto, and which will produce wrong results without filtering.
-->
The SELECT operations on Iceberg Tables with format version 2 do not read the delete files and remove the deleted rows as of now.

Please update the last section too.